homeassistant/components/vacuum/services.yaml

```diff
@@ -0,0 +1,131 @@
+turn_on:
+  description: Starts a new cleaning task.
+
+  fields:
+    entity_id:
+      description: Name of the botvac entity.
+      example: 'vacuum.xiaomi_vacuum_cleaner'
+
+turn_off:
+  description: Stop the current cleaning task and return to home.
+
+  fields:
+    entity_id:
+      description: Name of the botvac entity.
+      example: 'vacuum.xiaomi_vacuum_cleaner'
+
+stop:
+  description: Stop the current cleaning task.
+
+  fields:
+    entity_id:
+      description: Name of the botvac entity.
+      example: 'vacuum.xiaomi_vacuum_cleaner'
+
+locate:
+  description: Locate the vacuum cleaner robot.
+
+  fields:
+    entity_id:
+      description: Name of the botvac entity.
+      example: 'vacuum.xiaomi_vacuum_cleaner'
+
+start_pause:
+  description: Start, pause, or resume the cleaning task.
+
+  fields:
+    entity_id:
+      description: Name of the botvac entity.
+      example: 'vacuum.xiaomi_vacuum_cleaner'
+
+return_to_base:
+  description: Tells the vacuum cleaner to return to its dock.
```

Comment:
sorry, It sounds to me like "the service" _does_ something..., I'll try to remember this because I do that a lot. Thanks!